src/test/java/dev/jbang/BaseTest.java

```diff
@@ -0,0 +1,23 @@
+package dev.jbang;
+
+import java.io.IOException;
+
+import org.junit.Rule;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.rules.TemporaryFolder;
+
+public abstract class BaseTest {
+
+	@BeforeEach
+	void initEnv() throws IOException {
+		jbangTempDir.create();
+		environmentVariables.set("JBANG_DIR", jbangTempDir.getRoot().getPath());
```

Comment:
Isn't necessary to use `AfterAll` to set JBANG_DIR to its previous value?

Reply:
No need. It won't affect the value in the user's environment. And within the test run I also don't care because no test should ever use the original value anyway. And if a test uses a value from a previous test then that test is wrong and should be fixed  :-)